.github/workflows/static-checks.yaml

```diff
@@ -150,3 +150,36 @@ jobs:
     needs: skipper
     if: ${{ needs.skipper.outputs.skip_static != 'yes' }}
     uses: ./.github/workflows/govulncheck.yaml
+
+  codegen:
+    runs-on: ubuntu-22.04
+    needs: skipper
+    if: ${{ needs.skipper.outputs.skip_static != 'yes' }}
```

Comment:
@stevenhorsman I don't know whether this is correct or should rather be unconditional.

Reply:
Yes, thanks!